ribs.py

```diff
@@ -105,14 +108,45 @@ def key_pressed(key):
            (keycode not in last_frame_held_buttons)
 
 
+#
+# Simple sprite drawing
+#
+
+def draw_centered(img, pos, scale=(1., 1.), degrees=0):
+    """Draw img around position, scale the image and then rotate it in degrees before drawing."""
```

Comment:
Should we have some usage examples here?

```python
# Draw an image centered around 100, 100
draw_centered(teapot, (100, 100);

# Draw a stretched version of the image
draw_centered(teapot, (100, 100), scale = (1.5, 0.5)

# Draw a rotated version
draw_centered(teapot, (100, 100), degrees = 45)
```

Also, does the name of the function still make sense? Kind of I guess